src/UI/components/map/map.tsx

```diff
@@ -49,6 +49,17 @@ export const MapWrapper = () => {
 
   const [map, setMap] = useState<Map | null>(null);
 
+  function sleep(time: number) {
```

Comment:
Can this be moved to the `utils.ts` file? I can imagine we might need it elsewhere, and then you can test it :-)